.github/workflows/rust.yml

```diff
@@ -270,6 +270,7 @@ jobs:
           ockam --version
           echo $(which ockam)
           echo $BATS_TEST_RETRIES
+          python3 -m pip install --user uploadserver
```

Comment:
Should we add this [here](https://github.com/build-trust/.github/blob/custom-actions/actions/run_bats_test/action.yml#L121) too?

Reply:
@metaclips

Reply:
I added uploadserver to our Nix configuration @adrianbenavides

Reply:
I added uploadserver to our Nix configuration @adrianbenavides

Reply:
Thanks @metaclips 🙏